styleguide/_themes/derek/scss/patterns/blog-feed.scss

```diff
@@ -0,0 +1,71 @@
+@mixin blogFeed-textElements {
```

Comment:
Is this mixin going to be reused in the future?  If it's only referenced in one place, I don't necessarily see the need for a mixin, but if you have future plans for it, then that's okay with me.  Just checking.